### PR TITLE
feat(homepage): `<LearnSection />`

### PR DESCRIPTION
**Pull request by alexcarpenter**
## "Ready for Review" checklist

- [x] The Vercel preview link has been added to this PR's description
- [x] The Asana task has been added to this PR's description
- [x] This PR's link has been added to the "📐 Pull Request" field on the Asana task
- [x] This Vercel preview link has been added to the "🔍 Preview" field on the Asana task
- [x] The description template has been filled in below

---

## Relevant links

- [Preview link](https://dev-portal-git-achomepagelearn-section-hashicorp.vercel.app/) 🔎
- [Asana task](https://app.asana.com/0/1200641486648086/1202112386641568) 🎟️

## What

Implementing `<LearnSection />` to the homepage

CleanShot 2022-04-14 at 10 30 21@2x

## Why

New homepage implementation

### CODE REVIEW

File: src/views/homepage/components/learn-section/learn-section.module.css [hunk 1]
```diff
@@ -0,0 +1,81 @@
+.learnSection {
+  --container-max-width: 1240px;
+  --padding-y: 64px;
+  --padding-x: 24px;
+
```
Comment (alexcarpenter) [inline src/views/homepage/components/learn-section/learn-section.module.css:5]: Polish task to dial in these sizes across components https://app.asana.com/0/1202111817660946/1202127264881983

File: src/views/homepage/components/learn-section/learn-section.module.css [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  }
+}
+
+.introDescription {
+  margin: 12px 0 24px;
+
+  & a {
+    color: inherit;
+    text-decoration: underline;
+  }
```
Comment (alexcarpenter) [inline src/views/homepage/components/learn-section/learn-section.module.css:52]: Not sure if you all have a class or component that manages this styling for content.

Comment (ashleemboyer) [reply src/views/homepage/components/learn-section/learn-section.module.css:52]: Is this for the link that says "HashiCorp Cloud Engineer Certifications"? We have the `InlineLink` component that we use for content, but it has different styles than what you need here I think.

Comment (alexcarpenter) [reply src/views/homepage/components/learn-section/learn-section.module.css:52]: Went ahead and made use of InlineLink. Will figure out color in polish task https://app.asana.com/0/1200641486648086/1202127264881989

**Review by BrandonRomano — APPROVED** [review]
LGTM!  No major comments

**Pull request merged**